README.rst

```diff
@@ -57,6 +57,12 @@ integer.
                           picked randomly when encode.
     -e ENCODING, --encoding ENCODING
                           set encoding, only useful when base is not set
+                          
+
+For Example,
+
+::
```

Comment:
Could you change the **4** lines above to `For example::`?

Reply:
`.rst` is quite new for me.

Reply:
It was new for me at the time I wrote this README too :stuck_out_tongue:

I think you miss my request to remove the trailing newline (L61) and rebase the 2 commits together. It should be as easy as `git rebase -i HEAD~2` and `git push --force`.